test-curves/src/bls12_381/g1.rs

```diff
@@ -71,6 +77,34 @@ pub const G1_GENERATOR_X: Fq = MontFp!("3685416753713387016781088315183077757961
 #[rustfmt::skip]
 pub const G1_GENERATOR_Y: Fq = MontFp!("1339506544944476473020471379941921221584933875938349620426543736416511423956333506472724655353366534992391756441569");
 
+impl GLVConfig for Config {
+    const ENDO_COEFFS: &'static[Self::BaseField] = &[
+        MontFp!("793479390729215512621379701633421447060886740281060493010456487427281649075476305620758731620350")
+    ];
+
+    const LAMBDA: Self::ScalarField =
+        MontFp!("52435875175126190479447740508185965837461563690374988244538805122978187051009");
+
+    const SCALAR_DECOMP_COEFFS: [(bool, Self::ScalarField); 4] = [
+        (true, MontFp!("228988810152649578064853576960394133504")),
+        (true, MontFp!("1")),
+        (false, MontFp!("1")),
+        (true, MontFp!("228988810152649578064853576960394133503")),
+    ];
```

Comment:
Can we add some references indicating where these constants come from?

Reply:
Maybe @simonmasson has some scripts? Otherwise I can write some sage later

Reply:
@Pratyush  https://github.com/arkworks-rs/algebra/pull/644/commits/7d86856f6c6220caad5ad8eedda59057254d1143
It's using a different order than given for the bases than in the source, I might adapt the impl later to match the Guide also with parameter names, but first I'd like to get this merged so it could be tested against the curves implemented in https://github.com/arkworks-rs/curves/pull/147 without breaking anything.